templates/amp/_layout.html.twig

```diff
@@ -130,7 +130,8 @@
         <script type="application/json">
             {
                 "vars": {
-                    "account": "UA-72646335-1"
+                    "id": "GTM-TTW2FMQ",
```

Comment:
It's not here to add these vars. 
You have to change simply `<amp-analytics type="googleanalytics">` by `<amp-analytics config="https://www.googletagmanager.com/amp.json?id=GTM-TTW2FMQ&gtm.url=SOURCE_URL" data-credentials="include">`